crates/driver/openapi.yml

```diff
@@ -227,12 +227,14 @@ components:
       description: Response of the execute endpoint.
       type: object
       properties:
-        account:
-          description: The address that executes the transaction.
-          $ref: "#/components/schemas/Address"
-        nonce:
-          description: The nonce of the transaction.
-          type: integer
+        transactionIdentifier:
+          description: |
+            Extra calldata that is appended to the settlement contract invocation by the Solver to
+            identify the transaction. Chosen by the Solver.
```

Comment:
I think @MartinquaXD had decided to have this decided by the autopilot to avoid any issues with implementation-specific choices of identifiers that may lead to collision.

See https://github.com/cowprotocol/services/pull/1106#issuecomment-1400691371.

Reply:
I've moved it into the request (from the response) now. Also made it "arbitrary length" so the api doesn't specify that it has to be 4 bytes.